python-analysis-call-expressions/src/test/kotlin/org/jetbrains/research/ml/pythonAnalysis/CallExpressionsAnalysisExecutorTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.jetbrains.research.ml.pythonAnalysis
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.projectRoots.ProjectJdkTable
+import com.intellij.openapi.projectRoots.Sdk
+import com.intellij.openapi.roots.ProjectRootManager
+import org.jetbrains.research.ml.kotlinAnalysis.util.ParametrizedGoldenFileTest
+import org.jetbrains.research.ml.kotlinAnalysis.util.RepositoryOpenerUtil
+import org.jetbrains.research.pluginUtilities.sdk.PythonMockSdk
+import org.jetbrains.research.pluginUtilities.sdk.SdkConfigurer
+import org.jetbrains.research.pluginUtilities.util.Extension
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+@RunWith(Parameterized::class)
+open class CallExpressionsAnalysisExecutorTest : ParametrizedGoldenFileTest(
+    getResourcesRootPath(
+        ::CallExpressionsAnalysisExecutorTest,
+        "callExpressionsAnalysisExecutorData"
+    )
+) {
+    private lateinit var sdk: Sdk
+
+    companion object {
+        @JvmStatic
+        @Parameterized.Parameters(name = "{index}: ({0}, {1})")
+        fun getTestData() = getInAndOutArray(
+            ::CallExpressionsAnalysisExecutorTest,
+            "callExpressionsAnalysisExecutorData",
+            inExtension = Extension.EMPTY,
+            outExtension = Extension.CSV
+        )
+    }
+
+    private fun setupSdk(project: Project) {
+        val projectManager = ProjectRootManager.getInstance(project)
+        sdk = PythonMockSdk(testDataPath).create("3.8")
+        val sdkConfigurer = SdkConfigurer(project, projectManager)
+        sdkConfigurer.setProjectSdk(sdk)
+    }
+
+    override fun tearDown() {
+        ApplicationManager.getApplication().runWriteAction {
+            ProjectJdkTable.getInstance().removeJdk(sdk)
+        }
+        super.tearDown()
+    }
```

Comment:
Why did not you use tests with SDK from plugin utilities?

Reply:
I need to set up a mocked sdk to a test project. `setupSdk` from plugin utilities does not take the project as input, but set the sdk to `myFixture.project`.